docs/Opportunities/Hackathons/hackathons.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Hackathons
+tags:
+  - CAS
+  - Tandon
+  - Classes
+  - Clubs
+  - Social
+  - Student
+---
+
+# Hackathons
+
+If you're new to computer science, a hackathon is a great way to get your hands dirty on some real world projects that you are interested in. Imagine it as a sprint-like event where programmers, designers, and enthusiasts from various disciplines come together to create, build, and develop projects.
+
+# What
+
+A hackathon is an event where people engage in rapid and collaborative engineering over a relatively short period of time such as 24 or 48 hours. Participants form teams to work on a project, often with a specific theme or to solve a particular problem. It's a space where ideas are transformed into reality, where you're encouraged to think outside the box and push boundaries.
+
+# Popular Hackathons
+
+| Name                                                                                                      | Location        | Notes                                                                   |
+| --------------------------------------------------------------------------------------------------------- | --------------- | ----------------------------------------------------------------------- |
+| [HackNYU](https://hacknyu.org/)                                                                             | Global             | A 48-hour Global Hackathon Hosted by NYU |
```

Comment:
```suggestion
| [HackNYU](https://hacknyu.org/)                                                                             | New York, New York | A 48-hour Global Hackathon Hosted by NYU |
```
I think the location should be an actual location if the hackathon is in person, or "Online" if it is digital.

I know the hacknyu site says global hackathon but I think it makes more sense to put the actual location as New York instead. To be honest I don't really understand what global hackathon is supposed to mean. I think they just mean anyone in the world can participate (assuming they can travel to NYU)? But this is true for many hackathons.

Reply:
I thought it's global coz it might take place in other NYU campuses too like the one's in Abu Dhabit, ... But looking at the the HackNYU website, it says it's in NYC. Will change it